### ISSUE TITLE
Compression time is too slow

### ISSUE THREAD

**Issue by rovoskarl** [numandev1/react-native-compressor#302]
Great. This library is very useful. I found that it takes about 20 seconds to compress a large video of 100M, which is very bad for users.

**Comment by XChikuX** [comment]
I don't think this is really an issue. Have you compared it with other existing solutions?
Maybe try downloading an app on the appstore/playstore and check how long it takes to compress.

**Comment by HumedMohammed** [comment]
Having similar Issues here but for 90mb it is taking minutes like 5min avarage... is this a like predicted behavior or I am missing something... latest react native, expo bare, on a very latest mobile phone android version 12+
